app/views/errors/_overview.html.haml

```diff
@@ -66,4 +46,46 @@
           %dt Level
           %dd= @error.level
           %dt Logger
-          %dd= @error.platform
\ No newline at end of file
+          %dd= @error.platform
+  - valid_stacktrace = @issue.breadcrumbs_stacktrace != 'Missing breadcrumbs' && @issue.breadcrumbs_stacktrace.present?
```

Comment:
See this is really bad. This should not be like this. We should not check against the message.